avae/models.py

```diff
@@ -0,0 +1,98 @@
+import logging
+from abc import ABC, abstractmethod
+
+import torch
+import torch.nn as nn
+
+from avae.base import AbstractAffinityVAE
+
+from .base import SpatialDims
+
+
+def set_layer_dim(ndim):
```

Comment:
should we change the name of this function, to something indicating the selection of convolution dimension ?

Reply:
would `set_layer_conv_dim` work?